crates/sui-storage/src/fullnode_pending_tx_log.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! FullNodePendingTransactionLog is Write Ahead Log uesd in Transaction Orchestrator
```

Comment:
Write Ahead Log is not the right term here I think.
Also, this is tied to QD, not fullnode. You could totally have a FN that doesn't have QD/Orchestrator.

Reply:
renamed the struct and files to avoid link to a particular component.